node/primitives/bridge/src/lib.rs

```diff
@@ -65,6 +65,9 @@ pub const MAX_UNREWARDED_RELAYER_ENTRIES_AT_INBOUND_LANE: MessageNonce = 128;
 /// Maximal number of unconfirmed messages at inbound lane.
 pub const MAX_UNCONFIRMED_MESSAGES_AT_INBOUND_LANE: MessageNonce = 128;
 
+// 726f6c69
+pub const PANGORO_PANGOLIN_LANE: [u8; 4] = *b"roli";
```

Comment:
Might need companion changes in the bridger config:

https://github.com/darwinia-network/bridger/blob/96cf8321235bf0a0ed4b69e01a6e95e9dca42d4b/.maintain/config/task-pangolin-pangoro.toml#L11

Reply:
https://github.com/darwinia-network/bridger/pull/277